mautic/mautic
# Fixture landing pages emails

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #11471  



#### Description:



In the Doctrine fixtures, 2 emails and 1 landing page are created. Unfortunately, their data still relies on the old builder. Possibly a very old builder, as even disabling GrapesJS still needs to fix this.

When editing these pages or emails, clicking "build" brings up the old builder UI but is entirely broken. The console is full of errors.

I suggest updating the CSV files to create this content using the new structure.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Run bin/console `doctrine:fixtures:load`
3. Attempt to launch the builder for any of the created emails or page



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/12478"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
